src/gui/folderman.h

```diff
@@ -85,6 +85,7 @@ class FolderMan : public QObject
     Q_OBJECT
 public:
     static QString suggestSyncFolder(const QUrl &server, const QString &displayName);
+    static bool prepareFolder(const QString &folder);
```

Comment:
Do you ever intend to handle errors externally? Maybe `void` with some logging is more suitable?